src/modules/external/aha/components/AhaScore.tsx

```diff
@@ -102,12 +123,27 @@ const AhaScore = ({
         </Stack>
       )}
       <Stack direction='column' gap={1} alignItems='flex-start'>
-        <LabelWithContent label={item.text} helperText={item.helperText}>
+        <LabelWithContent
+          label={
+            // Custom solution for visually requiring this field.
+            // Form groups can't be required, but in the individual fields are hidden and/or readonly,
+            // so the normal RequiredLabel logic doesn't work automagically.
+            <RequiredLabel
+              text={item.text || 'AHA Score'}
+              required={true}
+              TypographyProps={{
+                fontWeight: 600,
+              }}
+            />
+          }
+          helperText={item.helperText}
+        >
           <LoadingButton
             loading={loading}
-            disabled={hasScore} // If value has already been fetched, disable the button
+            disabled={hasScore} // If value already exists, disable the button
```

Comment:
It looks like `hasScore` will be false if the value is `<0`, so does this mean the button will continually be enabled if the fetched score is -1? I think thats probably fine, just checking.

Reply:
Yes, this was my intention, to not prevent the user from re-fetching if there is no score. I'll add comments